docs/content/installation/csi_ug_install_operator_github.md

```diff
@@ -35,7 +35,7 @@ Use the following steps to install the operator and driver, with [GitHub](https:
     1.  Download the manifest from GitHub.
 
         ```
-        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.6.0/deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml > csi.ibm.com_v1_ibmblockcsi_cr.yaml
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml > csi.ibm.com_v1_ibmblockcsi_cr.yaml
```

Comment:
we changed the path of `csi.ibm.com_v1_ibmblockcsi_cr.yaml`, so it should be like this:
```suggestion
        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml > csi.ibm.com_v1_ibmblockcsi_cr.yaml
```